resilience4j-spring-cloud/src/main/java/io/github/resilience4j/bulkhead/autoconfigure/RefreshScopedBulkheadAutoConfiguration.java

```diff
@@ -11,7 +13,7 @@
 
 @Configuration
 @ConditionalOnClass({Bulkhead.class, RefreshScope.class})
-@AutoConfigureAfter(RefreshAutoConfiguration.class)
+@AutoConfigureAfter({CircuitBreakerAnnotationConfigScannerAutoConfiguration.class, RefreshAutoConfiguration.class})
```

Comment:
This is `BulkheadAutoConfiguration` not `CircuitBreaker`. Please remove `CircuitBreakerAnnotationConfigScannerAutoConfiguration`.